### DISCUSSION TITLE
bpi-r4 SD卡固件管理页面打不开

### DISCUSSION THREAD

**Discussion by wswcx** [immortalwrt/immortalwrt#1475] [General]
大佬们，在https://downloads.immortalwrt.org/snapshots/targets/mediatek/filogic/下载的
最新bananapi_bpi-r4-sdcard.img.gz固件，写好插到板子上开机能分配到IP地址，SSH也能正常接入，但是打不开管理页面，是啥问题

**Comment by 1715173329** [comment]
主线默认不带 luci
